components/list/test/list.test.js

```diff
@@ -85,10 +82,10 @@ describe('d2l-list', () => {
 	describe('navigation', () => {
 
 		[
-			{ keyPress: keyCodes.UP },
-			{ keyPress: keyCodes.DOWN }
+			{ keyPress: 'ArrowUp' },
+			{ keyPress: 'ArrowDown' }
```

Comment:
How come we can change this from a complex object to a string?

Reply:
Because previously we were simulating a key press using `KeyboardEvent` which requires a key "code" but we wanted a friendly name to put in the test name, so we needed two bits of information for each test. Now we're using the built-in Playwright stuff that uses the [standard string-based](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values) keys, so we only need the friendlier value.